applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_solver.py

```diff
@@ -255,8 +257,12 @@ def _CreateBuilderAndSolver(self):
         return KratosMultiphysics.ResidualBasedBlockBuilderAndSolver(linear_solver)
 
     def _CreateScheme(self):
-        # Fake scheme creation to do the solution update
-        scheme = KratosMultiphysics.ResidualBasedIncrementalUpdateStaticScheme()
+        if self.settings["formulation"]["element_type"].GetString() == "perturbation_transonic":
+            # Custom Scheme for transonic cases
+            scheme = KCPFApp.TransonicResidualBasedIncrementalUpdateStaticScheme(self.settings["transonic_scheme_settings"])
+        else:
+            # Fake scheme creation to do the solution update
+            scheme = KratosMultiphysics.ResidualBasedIncrementalUpdateStaticScheme()
```

Comment:
Not sure at all, but maybe makes sense to always use this scheme and do nothing if the element is not transonic... I've the feeling that in the long run this will give us less problems or unexpected surprises when switching elements (think in the ROM).

Reply:
Well, It makes sense to me, I could create a _PotentialFlowResidualBased........Scheme_ and pass it the element_type and the transonic settings, when it is not __perturbation_transonic_ It wouldn't do nothing as you say. It would be the same as now but at the C++ level.

Reply:
Something of this sort. In any case, lets discuss it next week at the office.

Reply:
Well, I've done some changes on this way, but ok, we'll talk about that next week.